web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -1,104 +1,85 @@
-                &deg;F
-              </div>
-            </div>
-            <p class="margin-0 font-body-3xs text-gray-50">{{ "Feels like" | t }}</p>
-            <div class="display-flex flex-fill margin-top-2">
-              <div class="flex-auto flex-align-center item margin-right-1">
-                {{ source(directory ~ "/assets/images/weather/icons/humidity.svg") }}
-              </div>
-              <div class="flex-fill flex-align-center">
-                {{ "Humidity: @humidity%" | t({ "@humidity": content["#data"]["humidity"] }) }}
-              </div>
-            </div>
-            <div class="display-flex flex-fill">
-              <div class="flex-auto flex-align-center item margin-right-1">
-                {{ source(directory ~ "/assets/images/weather/icons/wind.svg") }}
-              </div>
-              <div class="flex-align-center flex-auto">
-                {{ "Wind: @windSpeed mph @windDirection" | t({
-                  "@windSpeed": content["#data"]["wind"]["speed"],
-                  "@windDirection": content["#data"]["wind"]["shortDirection"]
-                  })
-                }}
-              </div>
-            </div>
-          </div>
-        </div>
-
-        {# Third row: generated narrative #}
-        <div class="grid-row margin-top-2">
-          <div class="grid-col-12">
-            {{ "The weather in <strong>@place</strong> is <strong>@conditions</strong>.
-                Temperature is <strong>@temperature&deg;F</strong>.
-                Humidity is <strong>@humidity%</strong>"
-                |
-                t({
-                  "@place": content["#data"]["location"],
-                  "@conditions": content["#data"]["conditions"]["long"] | lower,
-                  "@temperature": content["#data"]["temperature"],
-                  "@humidity": content["#data"]["humidity"]
-                })
-            }}{% if content["#data"]["wind"]["speed"] > 0 %}
-            {{
-              "with <strong>@windSpeed mph winds</strong> from the @windDirection"
-              |
-              t({
-                "@windSpeed": content["#data"]["wind"]["speed"],
-                "@windDirection": content["#data"]["wind"]["direction"]
-              })
-            }}{% endif %}.
-          </div>
+{# First row: widget-type interface #}
+<div class="grid-row margin-left-8 margin-y-4 weather-gov-current-conditions">
+  {# First half: icon, current temperature, and conditions text #}
+  <div class="tablet:grid-col-3 display-flex">
+    <div class="icon flex-align-center margin-right-1">
+      {{ source(directory ~ "/assets/images/weather/icons/conditions/" ~ content['#data']["icon"] )}}
+    </div>
+    <div class="flex-align-center">
+      <div class="font-body-3xl display-flex flex-fill">
+        {{ content["#data"]["temperature"] }}
+        <div class="margin-top-1 display-flex flex-align-start font-body-lg">
+          ℉
```

Comment:
Kudos to @claire-skies-noaa for this one! I didn't know there was a distinct character for this until [the hourly forecast PR](https://github.com/weather-gov/weather.gov/pull/395/files#diff-aeec8f47d377b7877360708a78349a5afde357c59a3a19aebcb9df6be993a195R32). So cool!